data/map.txt

```diff
@@ -30089,3 +30089,2613 @@ planet Zug
+		period 108.07588
+	object
+		sprite planet/desert4-b
+		distance 666
+		period 252.75656
+	object
+		sprite planet/rock15-b
+		distance 1046
+		period 497.49488
+	object
+		sprite planet/gas2-b
+		distance 3076
+		period 2508.8262
+		object
+			sprite planet/ice8
+			distance 299
+			period 195.43728
+		object
+			sprite planet/ice8-b
+			distance 466
+			period 380.25899
+	object
+		sprite planet/gas5
+		distance 9151
+		period 12873.419
+
+system 33b
+	pos 175.63468 9.0366981
+	government Uninhabited
+	link 3b
+	asteroids "small rock" 47 1.9095
+	asteroids "medium rock" 3 1.9095
+	asteroids "large rock" 6 2.1375
+	asteroids "small metal" 30 2.7645
+	asteroids "medium metal" 181 1.9665
+	asteroids "large metal" 6 2.565
+	minables tungsten 1 2.9962067
+	minables iron 31 3.5216762
+	haze _menu/haze-full
+	habitable 423.2
+	belt 1451
+	object
+		sprite star/f3
+		period 10
+	object
+		sprite planet/gas7
+		distance 2590
+		period 5730.8865
+	object
+		sprite planet/gas1-b
+		distance 5730
+		period 18858.368
+		object
+			sprite planet/dust4-b
+			distance 487
+			period 334.25047
+	object
+		sprite planet/jupiter
+		distance 8125
+		period 31842.504
+
+system 31b
+	pos 483.41245 63.036699
+	government Uninhabited
+	link 23b
+	asteroids "small rock" 25 0.671
+	asteroids "medium rock" 54 0.847
+	asteroids "large rock" 15 0.616
+	asteroids "small metal" 5 0.649
+	asteroids "medium metal" 3 1.045
+	asteroids "large metal" 4 0.99
+	minables aluminum 25 0.95568643
+	haze _menu/haze-full
+	habitable 28275.2
+	belt 1795
+	object
+		sprite star/m-giant
+		distance 1.4868643
+		period 10
+	object
+		sprite planet/cloud0
+		distance 4567.3464
+		period 151.58793
+
+system 34b
+	pos 280.07912 22.814476
+	government Uninhabited
+	link 21b
+	asteroids "large metal" 1 2.2344
+	haze _menu/haze-fullbright
+	habitable 6195.2
+	belt 1944
+	object
+		sprite star/g-giant
+		period 10
+	object
+		sprite planet/neptune-b
+		distance 578
+		period 157.9098
+	object
+		sprite planet/venus
+		distance 1263
+		period 510.06128
+	object
+		sprite planet/gas15-b
+		distance 2253
+		period 1215.2318
+	object
+		sprite planet/rock0
+		distance 2757
+		period 1645.0249
+
+planet "Giaru Gegno"
+	attributes "gegno quarg" quarg station
+	landscape land/station2
+	description "This Quarg Ringworld seems to be under construction, yet none of the Quarg here seem to be in any sort of busy rush. It doesn't look like construction was halted, but more slowed down and paced. Some of the hangers and docks on the outer part of the ring are oddly larger than others, and the inner quarters are wider and taller than normal."
+	spaceport "Even if it's under construction, there has to be at least billions of Quarg on this ring. They shift past you at a normal walking pace for a Quarg, and don't mind you much attention except from the occasional uninterested glare or two. There seems to be little conversation, and the silence is slightly unnerving to you as the only non-Quarg in sight."
```

Comment:
I don't think that the player would be the only non-Quarg here. Wouldn't there be some Gegno walking around as well? Or are the Gegno so hostile to the Quarg as to have no interest in visiting their ringworlds?
```suggestion
	spaceport "Even if it's under construction, there has to be at least billions of Quarg on this ring. They shift past you at a normal walking pace for a Quarg, and don't mind you much attention except from the occasional uninterested glance or two. There seems to be little conversation, and the silence is slightly unnerving to you as the only non-Quarg in sight."
```

Reply:
Yes, the intention is the Gegno do not want anything to do with the Quarg, and they do not want the Quarg to have anything to do with them. _However_, this is likely to change the later on the story goes, or perhaps they are on the ring, just unbeknownst to a passerby or a normal visitor (whatever normal would be for this example).

Reply:
That makes sense.